nst/models/baseline.py

```diff
@@ -76,11 +76,16 @@ def __init__(
         pass
 
     def training_step(self, batch, batch_idx):
-        # TODO extract content and style feature maps from _optimized_image using FeatureExtractor
-        # TODO compute content, style and total variation losses
-        # TODO compute and return weighted sum of losses
+        optimized_image_feature_maps = self._feature_extractor(self._optimized_image)
+
+        content_loss = self._content_loss(optimized_image_feature_maps[0])
+        style_loss = self._style_loss(optimized_image_feature_maps[1])
```

Comment:
Since _feature_extractor returns tuple, I would consider adding unpacking here for code clarity :)
```python
content_feautuer_maps, style_feature_maps = self._feature_extractor(self._optimized_image)
```

Reply:
Good point. Fixing immediately